taxi/tests.py

```diff
@@ -1,3 +1,139 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
 
-# Create your tests here.
+from taxi.forms import CarForm, DriverLicenseUpdateForm
+from taxi.models import Manufacturer, Driver, Car
+
+
+class ModelTests(TestCase):
+    def setUp(self):
+        self.manufacturer = Manufacturer.objects.create(
+            name="manufacturer",
+            country="country"
+        )
+        self.driver = get_user_model().objects.create(
+            username="driver",
+            password="password",
```

Comment:
Consider using `set_password()` method to ensure the password is hashed when creating a user.